adafruit_sht31d.py

```diff
@@ -57,18 +58,67 @@
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_SHT31D.git"
 
 
-SHT31_DEFAULT_ADDR = const(0x44)
-SHT31_MEAS_HIGHREP_STRETCH = const(0x2C06)
-SHT31_MEAS_MEDREP_STRETCH = const(0x2C0D)
-SHT31_MEAS_LOWREP_STRETCH = const(0x2C10)
-SHT31_MEAS_HIGHREP = const(0x2400)
-SHT31_MEAS_MEDREP = const(0x240B)
-SHT31_MEAS_LOWREP = const(0x2416)
-SHT31_READSTATUS = const(0xF32D)
-SHT31_CLEARSTATUS = const(0x3041)
-SHT31_SOFTRESET = const(0x30A2)
-SHT31_HEATEREN = const(0x306D)
-SHT31_HEATERDIS = const(0x3066)
+_SHT31_DEFAULT_ADDRESS = const(0x44)
+_SHT31_SECONDARY_ADDRESS = const(0x45)
+
+_SHT31_ADDRESSES = (_SHT31_DEFAULT_ADDRESS, _SHT31_SECONDARY_ADDRESS)
+
+_SHT31_READSERIALNBR = const(0x3780)
+_SHT31_READSTATUS = const(0xF32D)
+_SHT31_CLEARSTATUS = const(0x3041)
+_SHT31_HEATER_ENABLE = const(0x306D)
+_SHT31_HEATER_DISABLE = const(0x3066)
+_SHT31_SOFTRESET = const(0x30A2)
+_SHT31_NOSLEEP = const(0x303E)
+_SHT31_PDA_FETCH = const(0xE000)
+_SHT31_PDA_BREAK = const(0x3093)
+
+MODE_SSDA = 'Single'
+MODE_PDA = 'Periodic'
```

Comment:
Please rename these `MODE_SINGLE` and `MODE_PERIODIC` and then use them in the examples. Using strings prevents you from changing the value without breaking the API.

Reply:
I've renamed the variables and updated the example as requested. I've also included an example that exercises more of the single shot data acquisition mode features.